### PR TITLE
[IIIF-1099] Add version check profile

### PR DESCRIPTION
**Pull request by ksclarke**
* Add a build profile (and script) for checking Docker image package versions
* Update obsolete image package versions and application dependency versions
* Changed 'install's in README to 'verify's (which makes more sense since we're not installing to a local Maven repo)
* Parameterized versions in Dockerfile
* Extended timeouts on some of the longer running tests
* Remove stray Travis references

### CODE REVIEW

**Comment by ksclarke** [comment]
Had to update TestContainers to work around DockerHub's new rate limitations.

File: pom.xml [hunk 1]
```diff
@@ -1183,6 +1019,263 @@
...
+                    <jdwp.host.port>${jdwp.host.port}</jdwp.host.port>
+                    <jdwp.client.config>${jdwp.client.config}</jdwp.client.config>
+                  </systemPropertyVariables>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
```
Comment (markmatney) [inline pom.xml:1225]: I'm curious why this change

**Review by markmatney — DISMISSED** [review]
Just noticed a couple more `mvn install` references, and a question about the POM

**Review by DRickard — DISMISSED** [review]

File: pom.xml (hunk 1 above)
Comment (ksclarke) [reply pom.xml:1225]: Moving this code into a default profile (that's activated by the absence of a `-Dversions` argument, ensures these plugins don't run when we're just running the versions profile (that profile just uses a standard ubuntu image so the tests would fail against it).

**Review by DRickard — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (markmatney) [reply pom.xml:1225]: ah I see, that makes a lot of sense

**Review by markmatney — APPROVED** [review]
